### PR TITLE
Add multiple source support

### LINKED ISSUE #1480: Add support for multiple sources for each composition
Would require support to have multiple lines for one line in one source. Potentially could require two of our current lines being one line in alternately published source.

All our corrections are being done with published sources. It would be easier to simply make corrections / catalog differences between these sources by looking at the published works.

Our current sources will be "deprecated" as we move to figuring out all lines being from which published sources. Where we know our sources are based on certain publications we can set them now.

Related to https://github.com/ShabadOS/desktop/issues/410 where we will have incomplete published sources.

There will eventually need to be a dragger of sorts allowing for prioritization of these published sources

### PR DESCRIPTION
**Pull request by Harjot1Singh**
- Adds source support in data
- Rename existing sources to compilations
- Bundles source gurmukhi content in `line_content` table

Closes #1480

### CODE REVIEW

File: data/publications.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name_gurmukhi": "SRomxI gurduAwrw pRbMDk kmytI",
```
Comment (Sarabveer) [inline data/publications.json]: `SRo: gu: pR: kmytI`

File: data/publications.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "name_english": "Shiromani Gurdwara Parbandhak Committee"
```
Comment (Sarabveer) [inline data/publications.json]: `SGPC`

**Comment by Sarabveer** [comment]
Build is going to fail, need to add support for fallback variants/publishers/publications.

**Comment by Harjot1Singh** [comment]
Benchmarks fail, since benchmarking code has not been updated/adapted. Need to figure out how to do default priority sources with override support

**Pull request merged**